videoplayer/static/html/videoplayer_view.html

```diff
@@ -5,11 +5,16 @@
         }
     </style>
     <h2>{{ display_name }}</h2>
-    <video data-dashjs-player src="{{ url }}" controls disablePictureInPicture controlsList="nodownload" width="100%" height="100%"
-        html5_sources="[&quot;{{ url }}&quot;]">
-        <p class="vjs-no-js">To view this video please enable JavaScript, and consider upgrading to a web browser that
-            <a href="http://videojs.com/html5-video-support/" target="_blank">supports HTML5 video</a></p>
-    </video>
+
+    <video
+        id="video"
+        width="640"
+        poster="//shaka-player-demo.appspot.com/assets/poster.jpg"
```

Comment:
This is not right.  It will show some random poster when player is not playing.  Remove this poster image till we comeup with our own poster